systemtest/src/main/java/io/strimzi/systemtest/annotations/MultiNodeClusterOnly.java

```diff
@@ -15,4 +15,5 @@
 @Retention(RetentionPolicy.RUNTIME)
 @ExtendWith(MultiNodeClusterOnlyCondition.class)
 public @interface MultiNodeClusterOnly {
+    int workerNodeCount() default 1;
```

Comment:
Shouldn't be here higher number than 1?

Reply:
The condition is `current > expected` so it doesn't make sense

Reply:
Yeah, I saw there `current >= expected`, never mind 😄